nameable.rb

```diff
@@ -0,0 +1,30 @@
+class Nameable
+  def correct_name
+    raise NotImplementedError, 'Not Implmented'
+  end
+end
+
+class Decorator < Nameable
+  attr_accessor :nameable
+
+  def initialize(nameable)
+    @nameable = nameable
+    super()
+  end
+
+  def correct_name
+    @nameable.correct_name
+  end
+end
+
+class CapitalizeDecoration < Decorator
+  def correct_name
+    @nameable.correct_name
```

Comment:
- [ ] As the previous reviewer suggested kindly make sure the `correct_name` method in the `CapitalizeDecoration` class capitalizes the first letter of the given name because this is the purpose of that class